packages/core/src/browser/logger-frontend-contribution.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from 'inversify';
+import { CommandContribution, CommandRegistry, Command } from "../common/command";
+import { QuickOpenService, QuickOpenModel, QuickOpenItem, QuickOpenMode } from './quick-open';
+import { ILoggerServer, LoggerInfo } from '../common/logger-protocol';
+import { LogLevel, logLevelStrings, logLevelToString } from '../common/logger';
+
+/* Open a the quick open menu to choose a logger, then a log level.  */
+
+export const setLogLevelCommand: Command = {
+    id: 'setLogLevel',
+    label: 'Set Log Level'
+};
+
+/* Quick open item for the second step, when choosing a log level.  */
+
+class LogLevelQuickOpenItem extends QuickOpenItem {
```

Comment:
it would be nice to reexport items to let extension developers change default behaviour or workaround bugs

Reply:
What does "reexport" mean?

Reply:
sorry, i meant `export`

Reply:
Ok, and only the Item classes, not the Models?